Utils/github_workflow_scripts/github_workflow_scripts_tests/handle_external_pr_test.py

```diff
@@ -95,3 +96,30 @@ def test_get_packs_support_level_label_checkout_failed(mocker):
         assert get_packs_support_level_label(
             file_paths=['Packs/Pack1/pack_metadata.json'], external_pr_branch='test'
         ) == 'Xsoar Support Level'
+
+
+@pytest.mark.parametrize('pr_files,expected', [
+    (['Packs/HelloWorld/Layouts/layout-details-Hello_World_Alert-V2.json'], True),
```

Comment:
Can we add cases for all security review-required content items?

Reply:
@kgal-pan - Are we sure that `Packs/HelloWorld/{SECURITY_CONTENT_ITEMS[1]}/{SECURITY_CONTENT_ITEMS[1].lower()}-Hello_World_Alert-V2.json"` is the structure of the files directories? 
let's talk about it.